packages/trpc/server/routers/viewer/bookings/get.handler.ts

```diff
@@ -405,6 +416,12 @@ export async function getBookings({
         currency: booking.eventType?.currency || "usd",
         metadata: EventTypeMetaDataSchema.parse(booking.eventType?.metadata || {}),
       },
+      user: booking.user
+        ? {
+            ...booking.user,
+            isTeamAdminOrOwner,
+          }
```

Comment:
Are you sure this is the right place? booking user is the person assigned to the booking, we are checking Admin and Owner for the logged in user. This can cause confusion in future. Maybe create a separate field instead of clubbing it inside booking user.